students/rmart300/lesson04/dict_lab.py

```diff
@@ -0,0 +1,67 @@
+
+def dictionary1(person_dict):
+    """function to test basic functionality of dictionaries"""
+    print(person_dict)
+    person_dict.pop('cake')
+    print(person_dict)
+    person_dict['fruit']='Mango'
+    print(person_dict)
+    print('keys in dict')
+    for k in person_dict:
+        print(k)
+
+    print('values in dict')
+    for v in person_dict.values():
+        print(v)
+
+    print('Is cake a key in the dict? {}'.format(str('cake' in person_dict)))
```

Comment:
note that it is not necessary to cast to `str`, this will work just fine:
`print('Is cake a key in the dict? {}'.format('cake' in person_dict))`